src/themes/BootstrapTheme/Resources/views/modules/ZikulaUsersModule/User/main.html.twig

```diff
@@ -0,0 +1,26 @@
+{% set templatetitle = __('My account') %}
+{% include 'User/menu.html.twig' %}
+
+<ul class="list-group">
+{% for accountLink in accountLinks %}
+    <li class="list-group-item">
+        <a href="{{ accountLink.url }}">
+            <div class="media">
+                <div class="media-left">
+                {% if modvars.ZikulaUsersModule.accountdisplaygraphics == 1 %}
+                    {% if isset(accountLink.set) && !empty(accountLink.set) %}
+                        {% set iconset = accountLink.set %}
+                    {% else %}
+                        {% set iconset = 'null' %}
+                    {% endif %}
+                    {img src=$accountLink.icon modname=$accountLink.module set=$iconset class="media-object"}
```

Comment:
anything in a `/modules` dir is an override and not used until the module itself is ported to twig so, they can be left until then.